src/Storage/Storage.Management/help/Copy-AzStorageBlob.md

```diff
@@ -255,7 +255,9 @@ Accept wildcard characters: False
 ```
 
 ### -Force
-Force to overwrite the existing blob or file
+Answers 'yes' to any confirmation prompts.
```

Comment:
This is not right.
`-Force` will only suspress ShouldContinue, but will not suppress other prompt like ShouldProcess.